app.py

```diff
@@ -38,6 +38,17 @@
 cache = diskcache.Cache("./cache")
 background_callback_manager = DiskcacheManager(cache)
 
+# Fix Dash long callbacks crashing on macOS 10.13+ (also potentially not working
+# on other POSIX systems), caused by https://bugs.python.org/issue33725
+# (aka "beware of multithreaded process forking").
+#
+# Note: default start method has already been changed to "spawn" on darwin in
+# the `multiprocessing` library, but its fork, `multiprocess` still hasn't caught up.
+# (see docs: https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods)
```

Comment:
Instead of c/p this big comment around, you can also just point to https://github.com/dwave-examples/flow-shop-scheduling/pull/4.

Reply:
@randomir how's this? https://github.com/dwave-examples/mvrp/pull/13/commits/c56044207a2ce32c1b22a2ac9a6df79ed89fdd37

Reply:
Lovely, thanks!